src/secrets/actions/thunks.ts

```diff
@@ -112,15 +115,15 @@ export const deleteSecret = (id: string) => async (
     const resp = await apiDeleteSecret({
       orgID: org.id,
       data: {
-        secrets: [id],
+        secrets: [secret.id],
```

Comment:
did we ever resolve the API contract so that this only accepts an id and not an array of ids?

Reply:
No, who is the appropriate person to reach out to about that?

Reply:
I remember we had talked to @glinton about it, it might be worth touching base with him to see if we can clean that up ourselves

Reply:
Sorry, this slipped my mind, I've opened a pr (11243) that will allow ya'll to delete a single secret by id.

Reply:
You're the best Greg! Thanks!!

Reply:
Thanks Greg.  Works like a charm :)